### PR TITLE
Fixed Spelling bug #39191

### LINKED ISSUE #39191: Spelling bug
**Describe your problem and how to reproduce it:**

There is spelling mistake of "KERAS" instead of "KARAS" in first paragraph first row.

**Add a Link to the page with the problem:**
link: https://www.freecodecamp.org/learn/machine-learning-with-python/machine-learning-with-python-projects/cat-and-dog-image-classifier

**Tell us about your browser and operating system:**
* Browser Name: 
* Browser Version: 
* Operating System: 

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
image

### PR DESCRIPTION
**Pull request by yavidor**
Checklist:

- [x ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x] My pull request targets the `master` branch of freeCodeCamp.
- [ x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #39191 

I hope it's ok, it's my first time contributing to a project

### CODE REVIEW

**Review by ShaunSHamilton — APPROVED** [review]
Thank you, for getting this out. As a tip: Remember, it is best practice to work off of a new branch, and not the _master_ branch. Read more about it here: https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=making-changes-locally

This LTGM 👍 , but requires one more approval, before being merged.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Congrats @yavidor on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Comment by wk11-i** [comment]
wk11-i/yk2.co.th

**Pull request merged**
